src/Native/System.Native/pal_networking.cpp

```diff
@@ -624,3 +638,77 @@ SetIPv6Address(uint8_t* socketAddress, int32_t socketAddressLen, uint8_t* addres
 
     return PAL_SUCCESS;
 }
+
+extern "C" int32_t GetControlMessageBufferSize(int32_t isIPv4, int32_t isIPv6)
```

Comment:
Could you comment this? It looks a bit weird to see isIPv4 and isIPv6 since it seems like it should be one or the other